identity/src/keypair.rs

```diff
@@ -396,29 +392,29 @@ pub enum PublicKey {
     #[cfg(feature = "ed25519")]
     #[deprecated(
         since = "0.1.0",
-        note = "This enum will be made opaque in the future, use `PublicKey::into_ed25519` instead."
+        note = "This enum will be made opaque in the future, use `PublicKey::from` instead."
```

Comment:
This will need to mention both functions because only together, they fully replace the direct enum access.

Reply:
You mean like this?

`note = "This enum will be made opaque in the future, use `PublicKey::from` or `PublicKey::into_ed25519` instead."

Reply:
Yep!